tests/acceptance/features/apiOcm/share.feature

```diff
@@ -85,6 +87,10 @@ Feature: an user shares resources using ScienceMesh application
                     }
                   }
                 },
+                "eTag": {
+                  "type": "string",
+                  "pattern": "%etag_pattern%"
+                },
```

Comment:
should we also have “remoteItem” here?
put please "eTag" after "name" to remain consistent

Reply:
>should we also have “remoteItem” here?

IMO, we should

>put please "eTag" after "name" to remain consistent

done